libs/fetcher.js

```diff
@@ -23,7 +23,7 @@ function parseValue(value) {
         // the MAX_SAFE_INTEGER in javascript is 9007199254740991 which has a length of 16.
         // Using a length > 15 check is still safe and faster than trying to use another
         // library that supports big integers.
-        if (typeof ret === 'number' && value.length > 15) {
+        if (typeof ret === 'number' && String(value) !== String(ret)) {
```

Comment:
Yes, it seems that `isSafeInteger` is another solution. 
Up to now, I think `String(value)` is a simplest way to solve Infinity problem. Since it compares between original value and parsed value, it may also resolve other format problems, e.g. `JSON.parse('9007199254740991.11111')`